slnviz.py

```diff
@@ -335,7 +440,32 @@ def process(sln_file, dot_file, exclude, highlight, highlight_all, keep_deps):
     with open(dot_file, 'w') as f:
         f.write(txt)
 
-    print("Wrote output-file '{0}'.".format(dot_file))
+    log_info("Wrote output-file '{0}'.".format(dot_file))
+
+
+def set_style(theme, attributes):
+    global style_attributes
+
+    if not theme:
+        theme = 'dark'
+
+    debug("Using {0} theme".format(theme))
+    style_attributes = themes[theme]
+
+    if attributes:
+        for attr in attributes:
+            name, value = attr
+            if name not in style_attributes:
+                log_warning("Unknown style attribute defined: {0}".format(name))

```

Comment:
I would consider this an error, not a warning. Clearly the command has been invoked incorrectly?

Reply:
Was in doubt whether that is not too strong, but will modify. The rendering may otherwise hide an edge or node unexpectedly, leading to errors is human interpretation of the diagram.